src/rpdk/core/test.py

```diff
@@ -52,14 +54,61 @@ def temporary_ini_file():
         yield str(path)
 
 
-def get_overrides(root):
+def get_cloudformation_exports(region_name, endpoint_url):
+    session = create_sdk_session(region_name)
+    cfn_client = session.client("cloudformation", endpoint_url=endpoint_url)
+    next_token = ""
+    exports = {}
+    while next_token is not None:
+        result = (
+            cfn_client.list_exports(NextToken=next_token)
+            if next_token != ""
+            else cfn_client.list_exports()
+        )
+        exports = {
+            **exports,
+            **{export["Name"]: export["Value"] for export in result["Exports"]},
+        }
+        next_token = None if "NextToken" not in result else result["NextToken"]
+    return exports
+
+
+def render_jinja(overrides_raw, region_name, endpoint_url):
+    env = Environment(autoescape=True)
+    parsed_content = env.parse(json.dumps(overrides_raw))
+    variables = meta.find_undeclared_variables(parsed_content)
+    if variables:
+        exports = get_cloudformation_exports(region_name, endpoint_url)
+        invalid_exports = []
+        var_map = {}
+        for variable in variables:
```

Comment:
could compare keys of exports as set to variables using issuperset to avoid loop and concatenating list. set difference would be invalid exports

Reply:
Will do. Thought that jinja would throw an error for extra values you pass into render but it doesn't seem like it does